docs/usage.rst

```diff
@@ -271,19 +271,19 @@ Global Fatal Error Handler
 --------------------------
 
 The ``install`` method sets up a global listener for uncaught exceptions, and ``context`` and ``wrap`` can catch exceptions as well.
-These are situations where Raven catches what would otherwise be a fatal process-ending exception; a process should generally not
-continue to run after uncaught exceptions and similar such events (see `Node docs <http://nodejs.org/api/process.html#process_event_uncaughtexception>`_),
-so Raven has a concept of a "fatal error handler". When Raven catches what would otherwise be a fatal process-ending exception,
-it captures the exception and then calls the fatal error handler.
+These are situations where Raven catches what would otherwise be a fatal process-ending exception. A process should generally not
+continue to run after such events occur, (see `Node docs <http://nodejs.org/api/process.html#process_event_uncaughtexception>`_),
+so Raven has a concept of a "fatal error handler". When Raven catches an otherwise-fatal exception, it will capture the exception
```

Comment:
I think we should make it clear that we will capture *and send* the error, because someone might think they need to send it themselves in the fatal error handler